lib/system.js

```diff
@@ -68,7 +70,13 @@ module.exports = class SystemView extends ReadyResource {
   }
 
   async checkout (length) {
-    const checkout = new SystemView(this.core.session(), length)
+    const checkout = new SystemView(
```

Comment:
Note: this was added with the last commit (had missed this usage of new SystemView on my initial pass, so it wasn't yet passing through the maxCacheSize)

Reply:
what do you think about making the opt explicit here?

Reply:
checkout(length, { maxCacheSize = this.db.maxCacheSize } = {})

Reply:
Yeah, makes sense. See https://github.com/holepunchto/autobase/pull/156#discussion_r1650891447 though, we might be iterating on a method that's no longer relevant